server/app/uk/gov/ons/addressIndex/server/model/response/AddressResponse.scala

```diff
@@ -0,0 +1,251 @@
+      other.welshDependentLocality,
+      other.welshPostTown,
+      other.poBoxNumber,
+      other.startDate,
+      other.endDate
+    )),
+    nag = None,
+    geo = None,
+    underlyingScore = 1,
+    underlyingMaxScore = 1
+  )
+}
+
+
+/**
+  * Paf data on the address
+  *
+  * @param udprn                        udprn
+  * @param organisationName             organisation name
+  * @param departmentName               department name
+  * @param subBuildingName              sub building name
+  * @param buildingName                 building name
+  * @param buildingNumber               building number
+  * @param dependentThoroughfare        dependent thoroughfare
+  * @param thoroughfare                 thoroughfare
+  * @param doubleDependentLocality      double dependent locality
+  * @param dependentLocality            dependent locality
+  * @param postTown                     post town
+  * @param postcode                     postcode
+  * @param postcodeType                 postcode type
+  * @param deliveryPointSuffix          delivery point suffix
+  * @param welshDependentThoroughfare   welsh dependent thoroughfare
+  * @param welshThoroughfare            welsh thoroughfare
+  * @param welshDoubleDependentLocality welsh double dependent locality
+  * @param welshDependentLocality       welsh dependent locality
+  * @param welshPostTown                welsh post town
+  * @param poBoxNumber                  po box number
+  * @param startDate                    start date
+  * @param endDate                      end date
+  */
+case class AddressResponsePaf(
+  udprn: String,
+  organisationName: String,
+  departmentName: String,
+  subBuildingName: String,
+  buildingName: String,
+  buildingNumber: String,
+  dependentThoroughfare: String,
+  thoroughfare: String,
+  doubleDependentLocality: String,
+  dependentLocality: String,
+  postTown: String,
+  postcode: String,
+  postcodeType: String,
+  deliveryPointSuffix: String,
+  welshDependentThoroughfare: String,
+  welshThoroughfare: String,
+  welshDoubleDependentLocality: String,
+  welshDependentLocality: String,
+  welshPostTown: String,
+  poBoxNumber: String,
+  startDate: String,
+  endDate: String
+)
+
+/**
+  * Nag data on the address
+  *
+  * @param postcode postcode
+  */
+case class AddressResponseNag(
+  postcode: String
+)
+
+/**
+  * Contains address geo position
+  *
+  * @param latitude  latitude
+  * @param longitude longitude
+  * @param easting   easting
+  * @param northing  northing
+  */
+case class AddressResponseGeo(
+  latitude: Float,
+  longitude: Float,
+  easting: Int,
+  northing: Int
+)
+
+/**
+  * Contains response status
+  *
+  * @param code    http code
+  * @param message response description
+  */
+case class AddressResponseStatus(
+  code: Int,
+  message: String
+)
+
+/**
+  * Contains one response error
+  *
+  * @param code    error code
+  * @param message error description
+  */
+case class AddressResponseError(
```

Comment:
The gains would be quite nice actually. You could return that case object as the response if you extended it this way.